prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -1729,6 +1729,35 @@ ssize_t efa_rdm_ope_post_send(struct efa_rdm_ope *ope, int pkt_type)
 	return 0;
 }
 
+ssize_t efa_rdm_ope_post_send_handle_error(struct efa_rdm_ope *ope,
```

Comment:
add a doxgen function header to explain what this function does.

Reply:
Would not it makes more sense for `efa_rdm_post_send` to call this function directly upon failure?

Reply:
Sure, I can change it

Reply:
Fixed